ironfish/src/testUtilities/fixtures/account.ts

```diff
@@ -1,24 +1,35 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { Assert } from '../../assert'
 import { Account, AccountValue, Wallet } from '../../wallet'
+import { WithRequired } from '../types'
 import { FixtureGenerate, useFixture } from './fixture'
 
```

Comment:
The account fixtures should always have a `spendingKey`, so we are representing that here so that we can make the type system accept that.

Reply:
Note that this is almost a must since all old tests assumed account had a spending key. 50-100 tests would require addition of assertions if I didn't make this change. I can do that, or we can use `SpendingAccount`s for our fixtures